src/_interop/dtl.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-condition */
+
+import def, * as named from '@testing-library/dom'
```

Comment:
This seems to cause [some problems](https://github.com/testing-library/user-event/issues/1160) since there is no default export from `@testing-library/dom`.  I haven't seen this kind of approach before, just curious, what is the reason for writing it this way?